src/smart-components/role/add-role-permissions/add-role-permissions-summary-content.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, GridItem, Stack, StackItem, Text, TextContent, TextVariants, Title } from '@patternfly/react-core';
+
+const AddRolePermissionSummaryContent = ({ selectedPermissions, role }) => {
+  const { display_name: name, description } = role;
```

Comment:
Be carefull when desctructing objects without using `defaultProps` this could be `undefined` and could cause an error. You should either add `role` to `defaultProps` definition or when desctructing it use fallback `role || {}`

Reply:
You should resolve this one, either add default props or use default value.

Reply:
Hey! This was resolved actually, gave them ( = [], = {} ) respectively. Does it not reflect that in the code? or did you mean something other than that?

Reply:
Yeah, that's right!